assets/global.js

```diff
@@ -417,6 +418,20 @@ class HeaderDrawer extends MenuDrawer {
     trapFocus(this.mainDetailsToggle, summaryElement);
     document.body.classList.add(`overflow-hidden-${this.dataset.breakpoint}`);
   }
+
+  closeMenuDrawer(event, elementToFocus = false) {
+    if (event !== undefined) {
+      this.mainDetailsToggle.classList.remove('menu-opening');
+      this.mainDetailsToggle.querySelectorAll('details').forEach(details =>  {
+        details.removeAttribute('open');
+        details.classList.remove('menu-opening');
+      });
+      document.body.classList.remove(`overflow-hidden-${this.dataset.breakpoint}`);
+      removeTrapFocus(elementToFocus);
+      this.closeAnimation(this.mainDetailsToggle);
+      document.querySelector('#shopify-section-header').classList.remove('menu-open');
+    }
+  }
```

Comment:
I believe we just need to pass `elementToFocus` since that will be defined in the child instance of the method - whether it's the default (`false`) or something else.

Reply:
Nice catch, by the way.  I assumed there was a way to do this but never actually looked into it.  This is very useful.

Reply:
Yeah Lucas brought it up to me on the Slideshow PR and it was exactly what I needed there as well.